x-pack/plugins/secops/public/containers/overview/overview_network/index.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { getOr } from 'lodash/fp';
+import React from 'react';
+import { Query } from 'react-apollo';
+import { pure } from 'recompose';
+
+import { GetOverviewNetworkQuery, OverviewNetworkData } from '../../../graphql/types';
+import { createFilter } from '../../helpers';
+import { QueryTemplateProps } from '../../query_template';
+
+import { overviewNetworkQuery } from './index.gql_query';
+
+export interface OverviewNetworkArgs {
+  id: string;
+  overviewNetwork: OverviewNetworkData;
+}
+
+export interface OverviewNetworkProps extends QueryTemplateProps {
+  children: (args: OverviewNetworkArgs) => React.ReactNode;
+  sourceId: string;
+  endDate: number;
+  poll: number;
+  startDate: number;
+}
+
+export const OverviewNetworkQuery = pure<OverviewNetworkProps>(
+  ({ id = 'overviewNetworkQuery', children, filterQuery, sourceId, startDate, endDate }) => (
+    <Query<GetOverviewNetworkQuery.Query, GetOverviewNetworkQuery.Variables>
+      query={overviewNetworkQuery}
+      fetchPolicy="cache-and-network"
+      notifyOnNetworkStatusChange
+      variables={{
+        sourceId,
+        timerange: {
+          interval: '12h',
+          from: startDate!,
+          to: endDate!,
```

Comment:
You can remove the bangs on the end of these since you have excellent types and Typescript doesn't need type assertion turned off here.